### ISSUE TITLE
[iOS][Android] setAnalyticsCollectionEnabled is not working

### ISSUE THREAD

**Issue by Mighty683** [expo/expo#8264]
### Environment
  Expo CLI 3.17.24 environment info:
    System:
      OS: Linux 5.5 Manjaro Linux
      Shell: 5.0.16 - /bin/bash
    Binaries:
      Node: 13.13.0 - /usr/bin/node
      Yarn: 1.22.4 - ~/npm-global/bin/yarn
      npm: 6.14.1 - ~/npm-global/bin/npm
    IDEs:
      Android Studio: 3.6 AI-192.7142.36.36.6392135
    npmPackages:
      expo: ^37.0.0 => 37.0.7 
      react: 16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz => 0.61.4 

I am using expo flow and have this error both on iOS and Android version of app.

### Steps to Reproduce
import * as Analytics from 'expo-firebase-analytics';
Analytics.setAnalyticsCollectionEnabled(false);

### Expected Behavior

Set analytics collection to false.

### Actual Behavior

Throws Error:
```
[Unhandled promise rejection: Error: The method or property expo-firebase-analytics.setAnalyticsEnabled is not available on ios, are you sure you've linked all the native dependencies properly?]
* [native code]:null in construct
- node_modules/@babel/runtime/helpers/construct.js:19:26 in _construct
- node_modules/@babel/runtime/helpers/wrapNativeSuper.js:26:23 in Wrapper
* [native code]:null in construct
* http://192.168.8.117:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:99646:243 in <unknown>
- node_modules/@unimodules/react-native-adapter/build/errors/CodedError.js:8:8 in constructor
* [native code]:null in construct
* http://192.168.8.117:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:99689:243 in <unknown>
- node_modules/@unimodules/react-native-adapter/build/errors/UnavailabilityError.js:9:42 in constructor
- node_modules/expo-firebase-analytics/build/ExpoFirebaseAnalytics.js:14:42 in callAnalyticsModule
- node_modules/expo-firebase-analytics/build/ExpoFirebaseAnalytics.js:56:15 in <global>
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:44 in tryCatch
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:274:30 in invoke
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:44 in tryCatch
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:135:28 in invoke
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:170:17 in PromiseImpl$argument_0
- node_modules/promise/setimmediate/core.js:45:7 in tryCallTwo
- node_modules/promise/setimmediate/core.js:200:23 in doResolve
- node_modules/promise/setimmediate/core.js:66:12 in Promise
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:169:31 in PromiseImpl$argument_0
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:192:38 in enqueue
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:219:8 in exports.async
- node_modules/expo-firebase-analytics/build/ExpoFirebaseAnalytics.js:56:15 in <global>
- node_modules/expo-firebase-analytics/build/Analytics.js:58:7 in setAnalyticsCollectionEnabled
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:44 in tryCatch
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:274:30 in invoke
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:44 in tryCatch
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:135:28 in invoke
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:170:17 in PromiseImpl$argument_0
- node_modules/promise/setimmediate/core.js:45:7 in tryCallTwo
- node_modules/promise/setimmediate/core.js:200:23 in doResolve
- node_modules/promise/setimmediate/core.js:66:12 in Promise
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:169:31 in PromiseImpl$argument_0
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:192:38 in enqueue
- node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:219:8 in exports.async
- node_modules/expo-firebase-analytics/build/Analytics.js:58:22 in setAnalyticsCollectionEnabled
```

### Reproducible Demo

Can't provide working repro because working firebase integration is needed.

**Comment by byCedric** [comment]
Hi @Mighty683, it would help to still provide a repro but without the firebase credentials of course! This could be a repository on GitHub instead of a Snack. Without that, I won't be able to pinpoint or investigate the exact issue you experience. If you use the managed workflow, it would be good to double-check if you are not using anything outside the limitations [described here](https://docs.expo.io/versions/latest/sdk/firebase-analytics/#expo-client-limitations--configuration).

**Comment by EvanBacon** [comment]
Fixed in c4a8ef7
